readings.md

```diff
@@ -328,9 +328,7 @@ Use the Python interpreter to check your answers.
 Practice using the Python interpreter as a calculator:
 
 The volume of a sphere with radius r is 4/3 π r<sup>3</sup>. What is the volume of a sphere with
-radius 5? (Hint: 392.7 is wrong!) 
-
-Suppose the cover price of a book is $24.95, but bookstores get a
+radius 5? Hint: 392.7 is wrong! Suppose the cover price of a book is $24.95, but bookstores get a
```

Comment:
I'm not sure I understand why we're collapsing these two problems into a single paragraph. Perhaps convert this to a bullet list? Can you think of any other useful practice problems?

Reply:
Odd, your other PR separated these back out. Not sure which one is the real one.